core/base/elements-iteration-actions/assets.php

```diff
@@ -27,6 +27,7 @@ public function element_action( Element_Base $element_data ) {
 		$element_assets_depend = [
 			'styles' => $element_data->get_style_depends(),
 			'scripts' => $element_data->get_script_depends(),
+			'dynamic_imports' => $element_data->get_dynamic_import_depends(),
```

Comment:
```suggestion
			'dynamic_scripts' => $element_data->get_dynamic_scripts_depends(),
```

Maybe use this name to make sure developers understand it's a script.